src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -15,17 +15,31 @@
 use Nelmio\Alice\PersisterInterface;
 
 /**
- * The Doctrine persists the fixtures into an ObjectManager
+ * Adaptor for the ObjectManager to provide a Doctrine bridge.
+ *
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ * @author Robert Schönthal
  */
 class Doctrine implements PersisterInterface
 {
-    protected $om;
+    /**
+     * @var ObjectManager
+     */
+    protected $manager;
+
+    /**
+     * @var bool
+     */
     protected $flush;
 
-    public function __construct(ObjectManager $om, $doFlush = true)
+    /**
+     * @param ObjectManager $manager
+     * @param bool          $flush
+     */
+    public function __construct(ObjectManager $manager, $flush = true)
```

Comment:
I don't have an opinion on this - I personally haven't used the built-in persistence part of Alice. @Seldaek ?